CHANGELOG.md

```diff
@@ -1,6 +1,10 @@
 Changelog
 =========
 
+## [1.21.1] - 2021-03-13
+### Changed
+ * Change log level for connection create/close to debug.
```

Comment:
```suggestion
 * Change log level for connection create/close from info level to debug level to reduce noisy logs.
```

Reply:
(or something along these lines to provide a little more context)

Reply:
🤦 was too quick to click merge.......